static/app/bootstrap/renderDom.tsx

```diff
@@ -13,6 +14,17 @@ export function renderDom(
     return;
   }
 
-  const root = createRoot(rootEl);
-  root.render(<Component {...props} />);
+  // Types are for ConfigStore, the window object is from json and features is not a Set
+  if (
+    (window.__initialData.features as unknown as string[]).includes(
```

Comment:
Should we coerce to an array here jsut to be safe?

Also this should be typed on `Config` is it not?

Reply:
we talked in slack, not going to fix the type being reused for this